packages/asc-core/src/components/SearchBarStyle/SearchBarStyle.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from '../../styled-components'
+import InputStyle from '../InputStyle'
+import IconButtonStyle from '../IconButtonStyle'
+import { svgFill } from '../../utils'
+
+const SearchBarStyle = styled.div`
+  display: flex;
+
+  ${InputStyle} {
+    width: 500px;
```

Comment:
seems very specific? can't the input grow / shrink?

Reply:
right!